src/main/code/de/tu_chemnitz/se/exercise/codecharts/CodeChartsStringHandler.kt

```diff
@@ -0,0 +1,8 @@
+package de.tu_chemnitz.se.exercise.codecharts
+
+class CodeChartsStringHandler {
+  private val generatedStrings = setOf<String>()
+  fun generateStrings(Input: Int) {}
```

Comment:
Hier könnte man noch alle erlaubten Zeichen mit als Parameter übergeben. Oder in ner anderen Funktion.